packages/models-library/src/models_library/payments.py

```diff
@@ -1,11 +1,13 @@
 from decimal import Decimal
-from typing import Any, ClassVar
+from typing import Any, ClassVar, TypeAlias
 
 from models_library.emails import LowerCaseEmailStr
 from pydantic import BaseModel, Field, validator
 
 from .products import StripePriceID, StripeTaxRateID
 
+StripeInvoiceID: TypeAlias = str
```

Comment:
stupid question: why is this one not defined in the products like the above 2?

Reply:
My rationale behind is that the StripePriceID and StripeTaxRateID are directly connected to the Product (they are even in the products_prices table in DB). While StripeInvoiceID is something purely connected to Stripe and Stripe client will live in the Payment service.

Reply:
Ok, I will change in next pr that is coming